api/service/syncing/syncing.go

```diff
@@ -734,14 +734,15 @@ func (ss *StateSync) SyncLoop(bc *core.BlockChain, worker *worker.Worker, willJo
 		ss.RegisterNodeInfo()
 	}
 	ticker := time.NewTicker(SyncLoopFrequency * time.Second)
+Loop:
 	for {
 		select {
 		case <-ticker.C:
 			otherHeight := ss.getMaxPeerHeight()
 			currentHeight := bc.CurrentBlock().NumberU64()
 			if currentHeight >= otherHeight {
-				utils.Logger().Info().Msgf("[SYNC] Node is now IN SYNC! (ShardID: %d)", bc.ShardID())
-				break
+				utils.Logger().Info().Msgf("[SYNC] Node is now IN SYNC! (ShardID: %d, otherHeight: %d, currentHeight: %d)", bc.ShardID(), otherHeight, currentHeight)
```

Comment:
wont this create infinite loop? how does this fix the issue?

Reply:
This will break the infinite loop, before it just cannot escape the loop. right?